[KOGITO-877] - Improve the KOGITO images push-stating script to also … (#157)
kiegroup/kogito-images: scripts/README.md

Today we have these scripts:

> - [manage_kogito_version.py](manage_kogito_version.py)

Reviewer: why changing `-` with `_` ?
all other scripts are named with `-` ...

Author: Python doesn't allow to import modules with `-` in name. So had to change just for importing it.

Reviewer: the common script should solve this problem

```bash
$ pip install -U ruamel.yaml
```

> This script also imports `common.py` for updating version on the respective files

Reviewer: ```suggestion
This script has a dependency on `common.py`.
```

Staging images are the release candidates which are pushed mainly after big changes that has direct impact on how
the images will behave and also when new functionality is added.

> The script imports `common.py` for updating the version on:

Reviewer: ```suggestion
The script updates the version on:
```

 - all cekit modules
 - image.yaml file descriptor
 - kogito-imagestream.yaml
 

#### Script dependencies

The `push-staging.py` has a few dependencies that probably needs to be manually installed:

```bash
$ pip install -U docker yaml
$ pip install -U ruamel.yaml
```

This script is called as the last step of the `make push-staging` command defined on the [Makefile](../Makefile).

It will look for the current RC images available on [quay.io](https://quay.io/organization/kiegroup) to increase the rc tag 
accordingly then push the new tag so it can be tested by others. 

### Common script

> The `common.py` script defines some common functions used by `manage-kogito-version.py` and `push-staging.py`

Reviewer: ```suggestion
The `common.py` script defines some common functions for the scripts.
```
